breeze

```diff
@@ -2962,6 +2992,8 @@ breeze::setup_default_breeze_constants
 
 initialization::initialize_common_environment
 
+initialization::intialize_airflow_variables
```

Comment:
One thing here though. You will have to move this to inside of 

```
initialization::initialize_common_environment
```
This is an "umbrella initialization function that performs all the initialization together. all the other initializations are separated out for clarity, but they are actually called from `initialize_common_environment`

The CI scripts also run the `initialize_common_environment` and since the`intialize_airflow_variables` is not called there, you get the CI errors.

Reply:
That's the `initialize_common_environment` description: :)
```
# Common environment that is initialized by both Breeze and CI scripts
```